sdk/core/loader/types.h

```diff
@@ -862,9 +868,22 @@ namespace loader
 		static constexpr uint8_t InterruptStatusMask = uint8_t(0b11)
 		                                               << InterruptStatusShift;
 
+		/**
+		 * The flag indicating that this is a fake entry used to identify
+		 * sealing types.  Nothing should refer to this other than an import
+		 * table entry from the same compartment, which will be populated with
+		 * a sealing capability.
+		 */
```

Comment:
Is that right?  Don't the statically sealed objects also refer to the export table entry to specify what type they're sealed under?